fem/fespace.cpp

```diff
@@ -1105,12 +1064,14 @@ SparseMatrix* FiniteElementSpace::DerefinementMatrix(int old_ndofs,
                lR.GetRow(i, row);
                R->SetRow(r, old_vdofs, row);
                mark[m] = 1;
+               num_marked++;
             }
          }
       }
    }
 
-   MFEM_ASSERT(mark.Sum() == R->Height(), "Not all rows of R set.");
+   MFEM_VERIFY(num_marked == R->Height(),
```

Comment:
Isn't that equivalent to the old code?

Reply:
The main difference is that `MFEM_ASSERT` is replaced with `MFEM_VERIFY`. The computation of the sum on the fly is just a small optimization.